covin_slot_tracker.py

```diff
@@ -14,31 +17,34 @@ def getDate():
         Current date in DD-MM-YYYY format
 
     """
-    current_time = datetime.datetime.now()
-    day = current_time.day
-    month = current_time.month
-    year = current_time.year
-    date = "{dd}-{mm}-{yyyy}".format(dd=day,mm=month,yyyy=year)
+    today = datetime.date.today() + datetime.timedelta(days=i)
+    day = today.day
+    month = today.month
+    year = today.year
+    date = "{dd}-{mm}-{yyyy}".format(dd=day, mm=month, yyyy=year)
```

Comment:
This will give you i-th date from the current date. So when you ping the api, you are getting the slots for the i-th date